TcgPocket/Features/Cards/CardsController.cs

```diff
@@ -46,10 +46,11 @@ public async Task<ActionResult<Response<CardDetailDto>>> GetCardById([FromRoute]
         }
 
         [HttpGet]
-        public async Task<ActionResult<Response<PagedResult<CardGetDto>>>> GetAllCardsPaginatedQuery([FromQuery]PageDto data)
+        public async Task<ActionResult<Response<PagedResult<CardGetDto>>>> 
```

Comment:
```suggestion
        public async Task<ActionResult<Response<PagedResult<CardDetailDto>>>> 
```
Rec. changing here and inside of the GetPagedCardsQuery so that card attributes are included